doc/whats_new/upcoming_changes/sklearn.utils/27818.fix.rst

```diff
@@ -0,0 +1,6 @@
+- Numerical stabibilty (mostly overflows) of the Yeo-Johnson transform with
+  `PowerTransformer(method="yeo-johnson")` is fixed if scipy verion >= 1.9 is used.
+  By :user: `Sylvain Marié<@smarie>`, `Mohamed yaich<@yaichm>`,
+  `Oussama Er-rabie<@eroussama>`, `Mohamed Dlimi<@Dlimim>`,
+  `Hamza Zeroual<@HamzaLuffy>`, `Amine Hannoun<@AmineHannoun>` and
+  `Xuefeng Xu <@xuefeng-xu>`.
```

Comment:
The whatsnew should be placed in `preprocessing` as `PowerTransformer` has the user facing change.